Python/App.py

```diff
@@ -794,16 +802,37 @@ def returns():
             print("Could not retrieve specified Returnment Entity")
         finally:
             client.close()
-    elif 'issue_info' in request.form: # case added for issues and file upload - Sameer
+    # Sameer: Code start.
+    elif 'issue_info' in request.form:
         issueInfo = request.form['issue_info']
         location = app.config['UPLOAD_FOLDER']
         command = f'echo "{issueInfo}" > {location}issueId.mail'
         print(command)
         os.system(command) #create a issue file.
+        query = f"insert into CustomerIssue values (\"{issueInfo}\",\"This is a issue\""
         if 'file' in request.files:
             print(request.files)
             file = request.files['file']
-            file.save(os.path.join(app.config['UPLOAD_FOLDER'], file.filename))
+            loc = os.path.join(app.config['UPLOAD_FOLDER'], file.filename)
+            file.save(loc)
+            # if file type is zip then unzip that file
+            if get_file_ext(loc) == "zip":
+                unzipfile.unzip(loc, app.config['UPLOAD_FOLDER'])
+            query = query + f",LOAD_FILE(\'{loc}\'));"
+        else:
+            query = ");"
+        # store in database
+        client = pymysql.connect(host='localhost', user="root", password="", database="eCommerce01")
+        try:
+            print(query)
+            cursor = client.cursor()
+            cursor.execute(query)
+            client.commit()
+        except Exception:
```

Comment:
Can you print the exception message also? 
```except Exception e: 
   print(e)
```

Reply:
sure.